src/test/java/org/broadinstitute/consent/http/AbstractTest.java

```diff
@@ -247,4 +251,57 @@ protected DBI getApplicationJdbi() {
         return new DBIFactory().build(environment, configuration.getDataSourceFactory(), "mysql");
     }
 
+    String consentPath() {
```

Comment:
All of these utility methods are copied from what was `ConsentServiceTest`. That class now actually tests consent service methods instead of being just a group of utility methods.